osu.Game/Screens/Play/HUD/LegacyComboCounter.cs

```diff
@@ -45,6 +45,13 @@ public class LegacyComboCounter : CompositeDrawable, ISkinnableDrawable
         [Resolved]
         private ISkinSource skin { get; set; }
 
+        private readonly Container counterContainer;
+
+        public bool ContentVisible
+        {
+            set => counterContainer.Alpha = value ? 1 : 0;
+        }
```

Comment:
Well, the usage would be enough to explain that, I wasn't sure about duplicating comments between here and where it's used ("this is used to hide legacy HUD combo counter for rulesets that provide their own"), so I couldn't really think of a reason to add one.

Reply:
One reason is I imagine that the same thing will be used by mania and taiko once those ruleset-specific counters make it in. Unless there's a better system in place to do this by then.

Reply:
Ah right, thought this is osu!catch-only.

Reply:
A few questions here:

- Why was this marked resolved?
- The comment mentions this is "temporarily done". What is the permanent solution that you see?

Reply:
- I've added a xmldoc for it now, so this should be considered resolved by now.
- I actually got carried away by that since the initial changes of the PR, I can't say I foresee a permanent solution for this, will remove that.

Reply:
What about the fact that this is going to also occur in other rulesets? Are you planning on making the same local workaround / comment in each location?

Reply:
I guess? I'm not really sure if that matters with this PR.

Reply:
So how would you see the combo display in mania being done? In the same way? With another `ContentVisible` bool?

Reply:
I guess it's the same class at the end of the day. How about calling the variable something like "HiddenByRulesetImplementation" or similar?

Reply:
Sounds good.